hircluster.c

```diff
@@ -1848,6 +1891,23 @@ int redisClusterSetOptionConnectNonBlock(redisClusterContext *cc) {
     return REDIS_OK;
 }
 
+/**
+ * Configure a password used when connecting to password-protected
+ * Redis instances. (See Redis AUTH command)
+ */
+int redisClusterSetOptionPassword(redisClusterContext *cc,
+                                  const char *password) {
+
+    if (cc == NULL || password == NULL) {
+        return REDIS_ERR;
+    }
+
+    strncpy(cc->password, password, sizeof(cc->password) - 1);
+    cc->password[sizeof(cc->password) - 1] = '\0';
```

Comment:
A truncated password will probably not work. Why not return an error if the password is too long?

Of course, memory safety is more important, so setting the last byte to \0 is probably still a good idea.

Reply:
Done